### ISSUE TITLE
OCR subproject

### ISSUE THREAD

**Issue by congdon** [OpenMaine/campaign-finance#8]
Noting this here that a new subproject has been started in github to do optical character recognition.

**Comment by congdon** [comment]
And also that the goal will be to be able to take either the PDFs from the PPH (hoping they do that again) or cellphone images to reduce the need for humans to type in info.

**Comment by mdrideout** [comment]
Hey, just want to be sure no one is depending on this for a deadline but I'm giving it a shot.

**Comment by congdon** [comment]
Nope, no one is depending on it; we are just getting done what we can for this election. We did not expect to have such a hard time finding typists, since there was a willing band of them for the last election (but we never got anything online for that one).

**Comment by congdon** [comment]
If it works out on your end, though: Please keep me posted on your path; I might be able to help. Also, once/if you chart out a development plan, we can add the 'help wanted' flag if you like.
